webapp/channels/src/components/post/post_component.tsx

```diff
@@ -390,12 +390,12 @@ const PostComponent = (props: Props): JSX.Element => {
     }, [post, props.actions, props.actions.selectPostFromRightHandSideSearch]);
 
     const handleThreadClick = useCallback((e: React.MouseEvent) => {
-        if (props.currentTeam.id === props.team?.id) {
+        if (props.currentTeam.id === teamId) {
             handleCommentClick(e);
         } else {
             handleJumpClick(e);
         }
-    }, [handleCommentClick, handleJumpClick]);
+    }, [handleCommentClick, handleJumpClick, props.currentTeam.id, teamId]);
```

Comment:
is `props.currentTeam.id` necessary?

Reply:
I don't think so, line 130 defines teamId as `const teamId = props.team?.id || props.currentTeam.id;`

Reply:
If `teamId` is always `props.team?.id`, I believe `props.currentTeam.id` could become stale in the callback. Even if it wasn't needed—in other cases we're explicit e.g. we include `dispatch` in deps when it doesn't usually change in our case, plus `react-hooks/exhaustive-deps` would complain about it if it were on.